src/basic/string-util.h

```diff
@@ -32,6 +32,12 @@ static inline bool streq_ptr(const char *a, const char *b) {
         return strcmp_ptr(a, b) == 0;
 }
 
+static inline char* strstr_ptr(const char *haystack, const char *needle) {
```

Comment:
This function is not used at the moment. OTOH, we might need it for future CI fixes--it's only used in tests so far--so let's get it in...